atomic_reactor/plugins/pre_add_help.py

```diff
@@ -45,13 +48,17 @@ def __init__(self, tasker, workflow, help_file=DEFAULT_HELP_FILENAME):
     def run(self):
         """
         run the plugin
+
+        The plugin returns None if exception occurred,
```

Comment:
Updated the code: now it returns a dict with `help_file` and a numeric status (so that `koji_promote` can easily detemine the result)